db/fixtures/001_badges.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+badge_grouping = BadgeGrouping.find_by(name: "Community")
```

Comment:
This can be done in one line since `find_by` returns `nil` when it finds nothing

```ruby
return unless badge_grouping = BadgeGrouping.find_by(name: "Community")
```